README.md

```diff
@@ -22,6 +22,11 @@ automatically added to out-going jdbc calls if the jdbc Streaming Expression doe
 
 Commands:
 
+#### help
+Display help text in the note window. Command-specific help is provided if a valid command is provided as an argument.
+
+Usage: `help [{specific-command}]`
```

Comment:
is this literally like `help [search]` ???      I poked around a bit, just to see if this is a convention in the Zeppelin world, but it doesn't appear to be.  In fact, there doesn't appear to be any convention on the various interpreters of having a standard entry point to help!   Some do support the keyword `help` and some don't!.   I think `help search` rolls off the keyboard easier ;-)

Reply:
so maybe just `help {specific-command}` ?  I.e remove the square brackets?

Reply:
Ah, so that's what raised the question.

Hmm, I was trying to stick to the convention in man-pages and help text where optional arguments are put in brackets, but it does seem like a little much here maybe.  Will remove.